solana/bridge_token_factory/programs/bridge_token_factory/src/instructions/user/init_transfer.rs

```diff
@@ -50,17 +50,17 @@ pub struct InitTransfer<'info> {
     #[account(
         mut,
         seeds = [SOL_VAULT_SEED],
-        bump = wormhole.config.bumps.sol_vault,
+        bump = common.config.bumps.sol_vault,
     )]
     pub sol_vault: SystemAccount<'info>,
 
     #[account(
         mut,
-        owner = wormhole.system_program.key(),
+        owner = common.system_program.key(),
```

Comment:
If I understood correctly, in issue 9, it is suggested to delete this line entirely as unnecessary.

https://github.com/AuditOneAuditReviews/Omnibridge2-audit-review/issues/9

Reply:
Ah, I think I referred to 9 by mistake. I did not intend to fix that one